cheddar/cheddar-rest/src/main/java/com/clicktravel/cheddar/rest/media/MediaTypes.java

```diff
@@ -16,8 +16,12 @@
  */
 package com.clicktravel.cheddar.rest.media;
 
+import javax.ws.rs.core.MediaType;
+
 public class MediaTypes {
 
     public static final String CDM_V1_JSON = "application/vnd.clicktravel.schema-v1+json";
```

Comment:
I had this thought also, but realised this would add a dependency to JAX-RS API, which may not be desirable to all CDM consumers.

Since raising this pull request, I realised I couldn't achieve what I needed in my application with this change in place. I've instead gone a different route and copied the MediaType into my application code. I agree that this change doesn't belong in Cheddar itself.